src/components/PersonInformationForm/PersonInformationForm.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { Form } from 'react-bootstrap'
+import { FormWrapper } from './PersonInformationForm.style'
+
+const PersonInformationForm = () => {
+  const [personInfo, setPersonInfo] = useState({
+    gender: 'Male',
+    age: 25,
+    height: 175,
+    weight: 70,
+  })
+
+  const genders = ['Male', 'Female']
+
+  const handleOnlyNumbers = (input) => {
+    let number = input.target.value
+    console.log(number)
+
+    if (!Number(number)) {
```

Comment:
Not sure but I think you can just convert it to number so this condition won't be needed